journal/src/main/scala/com/evolutiongaming/kafka/journal/Journal.scala

```diff
@@ -16,13 +18,16 @@ import scala.concurrent.duration._
 
 trait Journal[F[_]] {
 
-  def append(
-    events: Nel[Event],
+  def append[A](
```

Comment:
Is it possible to move `A` type parameter to `Journal[F, A]`?

Reply:
It is possible, yes, but it was decided to place it here per discussions with @t3hnar